src/freertos_drivers/esp_idf/EspIdfWiFi.cxx

```diff
@@ -1146,30 +765,29 @@ void EspIdfWiFiBase::ip_event_handler(
             ip_event_got_ip_t *d = static_cast<ip_event_got_ip_t *>(data);
             HASSERT(ipAcquiredSta_ == false);
             ipAcquiredSta_ = true;
-            mdns_restore_sta();
-            {
-                OSMutexLock locker(&lock_);
-                if (mdnsClientCache_.size())
-                {
-                    // Trigger an immediate scan in order to refresh the cache.
-                    mdns_scanning_start_or_trigger_refresh();
-                }
-            }
             char ip_addr[16];
```

Comment:
does this need to be increased from 16 to 17 to account for the trailing null?
or a comment to indicate why not to increase.